host/xtest/regression_1000.c

```diff
@@ -2211,15 +2212,21 @@ static void xtest_tee_test_1029(ADBG_Case_t *c)
 		return;
 
 	Do_ADBG_BeginSubCase(c, "TLS variables (main program)");
-	ADBG_EXPECT_TEEC_SUCCESS(c,
-		TEEC_InvokeCommand(&session, TA_OS_TEST_CMD_TLS_TEST_MAIN, NULL,
-				   &ret_orig));
+	res = TEEC_InvokeCommand(&session, TA_OS_TEST_CMD_TLS_TEST_MAIN, NULL,
+				 &ret_orig);
+	if (res == TEEC_ERROR_NOT_SUPPORTED)
+		printf("TA returned TEEC_ERROR_NOT_SUPPORTED\n");
```

Comment:
`Do_ADBG_Log()` instead of `printf()`.

For consistency with other skipped tested:
`Do_ADBG_Log(" - 1029 -   skip test, Thread Local Storage test not supported");`

Reply:
OK, I will use `Do_ADBG_Log()` and adjust the format for more consistency. But I'd rather not repeat "Thread Local Storage" which is printed immediately before. It's better to inform the user that the cause is with the TA I think.